databases/percona/Portfile

```diff
@@ -130,7 +130,7 @@ if {$subport eq $name} {
                         -DENABLE_DTRACE:BOOL=OFF \
                         -DWITH_EDITLINE:STRING=bundled \
                         -DWITH_LIBWRAP:BOOL=ON \
-                        -DWITH_SSL:STRING=bundled \
+                        -DWITH_SSL:STRING=${prefix} \
```

Comment:
Should this instead be `-DWITH_SSL:PATH="${prefix}"` or does it not matter?

Reply:
Yes, it's `${prefix}` in `mysql8`.

Reply:
Both quoting and the use of `STRING` vs `PATH` (I do not know much at all about CMake). There is inconsistency among MySQL ports or various options in them.

Reply:
Disclaimer: I am not a CMake expert either.

Basically CMake variables can have types, like `PATH`, `STRING`, or `BOOL`.  When in doubt, just specify the option and value without the `:<TYPE>` suffix.  The types themselves look like they are simply used to help CMake create the right graphical widgets when configuring using CMake's graphical tools: https://cmake.org/cmake/help/v3.0/command/set.html

If you're curious about what a particular variable's type is, you can dig into the files in the `cmake` directory, to see whether the variable is set using `OPTION()` or `SET()`.  If `OPTION`, then the variable's type is `BOOL`.  Else look for where the variable is being set using `SET`.  You may see it as `SET(<var> ... CACHE <type> ...)`.  If you see `SET(<var> ... INTERNAL ...)`, then that variable's type is `STRING`.

`CACHE` variables and `OPTION`s are the kind of variables CMake exposes on the command line, and which we would be setting in a `Portfile`.

Here are some examples from `percona-server` source:
* `WITH_SSL` set in `CACHE`: https://github.com/percona/percona-server/blob/41fa622c3399b68b77104054673fab49b92c3fb8/cmake/ssl.cmake#L98
* `WITH_LIBWRAP` set using `OPTION`: https://github.com/percona/percona-server/blob/41fa622c3399b68b77104054673fab49b92c3fb8/configure.cmake#L144